packages/synthetix-governance/contracts/storage/DebtShareStorage.sol

```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/IDebtShare.sol";
+import "@synthetixio/core-contracts/contracts/utils/SetUtil.sol";
+
+contract DebtShareStorage {
+    struct DebtShareStore {
+        // Debt share contract used to determine vote power
+        IDebtShare debtShareContract;
+        // Debt share snapshot id by epoch index
+        mapping(uint => uint128) debtShareIds;
```

Comment:
if epoch only increments, wouldn't it be simpler to just use an array?

Reply:
Yeah totally, I'll create a separate ticket for changing this to an array.

Reply:
Done.
https://github.com/Synthetixio/synthetix-v3/issues/916